packages/now-go/index.js

```diff
@@ -108,10 +108,20 @@ async function build({ files, entrypoint }) {
 
     // move user go file to folder
     try {
-      await move(
-        downloadedFiles[entrypoint].fsPath,
-        `${join(entrypointDirname, packageName, entrypoint)}`,
-      );
+      // default path
+      let finalDestination = join(entrypointDirname, packageName, entrypoint);
+      const entrypointArr = entrypoint.split('/');
```

Comment:
```suggestion
      const entrypointArr = entrypoint.split(sep);
```

I think we should support any filesystem

Reply:
I can't offer the suggestion but on Line 1 will need to be `const { join, sep, dirname } = require('path');`